### DISCUSSION TITLE
aircraft.ts needs cleanup and additional information

### DISCUSSION THREAD

**Discussion by chris-pryazhentsev** [johanohly/AirTrail#20] [General]
I'd just like to say that I really like AirTrail and that the app has been a pleasure to use. Being extremely nitpicky however I have noticed that the aircraft data in aircraft.ts needs some work, for example the file is not alphabetized making it difficult to read, as well, not all information is complete for example:

* Inconsistencies between naming (ex. A330 vs A330-300)
* lack of certain aircraft type (ex. A330-800/-900, 777-300ER, 747-8i, ERJ-130,ERJ1-140)
* weight classes needing some work, I would not consider an ERJ-145 in the same weight class as a 737 

I am willing to do this work but just wanted to check before I put in a PR if this would make sense to work on re-roganizing the data and providing more complete aircraft data within the file.

Thanks!

**Comment by johanohly** [comment]
I am very happy to hear it! 

You are more than welcome to make a PR revising the aircraft list. The main reasons it ended up so messy is that
1. I removed duplicate ICAO-codes because that is what I store in the database so it has to be unique. In the process I almost certainly made some wrong calls on which entry to keep (to keep each entry as generic as possible)
2. I didn't even consider sorting it - that makes perfect sense though, please do

Regarding the WTCs, unless I have missed something, the two are in the same category...

**Reply by chris-pryazhentsev** [reply]
[PR Logged](https://github.com/johanohly/AirTrail/pull/21)
